docs/content/preview/deploy/public-clouds/aws/manual-deployment.md

```diff
@@ -316,8 +316,8 @@ for ip in $ALL_NODES; do \
    echo =======$ip=======; \
    ssh -i $PEM $ADMIN_USER@$ip \
       "cd ~/yb-software; \
-       curl -k -o yugabyte-${YB_VERSION}-linux.tar.gz \
-         https://downloads.yugabyte.com/yugabyte-${YB_VERSION}-linux.tar.gz"; \
+       curl -k -o -L yugabyte-${YB_VERSION}-linux.tar.gz \
```

Comment:
This should maybe be:
```
curl -OL https://software.yugabyte.com/yugabyte-${YB_VERSION}-linux.tar.gz
```

The `-k` option means not to check the SSL cert, but we have valid SSL certs.
The `-O` option means name the downloaded file the same as it is on the remote side which is `yugabyte-${YB_VERSION}-linux.tar.gz`